src/main/java/org/tarantool/jdbc/ds/SQLDataSource.java

```diff
@@ -0,0 +1,163 @@
+        logWriter = out;
+    }
+
+    @Override
+    public void setLoginTimeout(int seconds) {
+        SQLProperty.INIT_TIMEOUT.setInt(properties, (int) TimeUnit.SECONDS.toMillis(seconds));
+    }
+
+    @Override
+    public int getLoginTimeout() throws SQLException {
+        return (int) TimeUnit.MILLISECONDS.toSeconds(SQLProperty.INIT_TIMEOUT.getInt(properties));
+    }
+
+    @Override
+    public Logger getParentLogger() throws SQLFeatureNotSupportedException {
+        throw new SQLFeatureNotSupportedException();
+    }
+
+    @Override
+    public <T> T unwrap(Class<T> type) throws SQLException {
+        if (isWrapperFor(type)) {
+            return type.cast(this);
+        }
+        throw new SQLNonTransientException("Statement does not wrap " + type.getName());
+    }
+
+    @Override
+    public boolean isWrapperFor(Class<?> type) {
+        return type.isAssignableFrom(this.getClass());
+    }
+
+    @Override
+    public String getServerName() {
+        return SQLProperty.HOST.getString(properties);
+    }
+
+    @Override
+    public void setServerName(String serverName) {
+        SQLProperty.HOST.setString(properties, serverName);
+    }
+
+    @Override
+    public int getPortNumber() throws SQLException {
+        return SQLProperty.PORT.getInt(properties);
+    }
+
+    @Override
+    public void setPortNumber(int port) {
+        SQLProperty.PORT.setInt(properties, port);
+    }
+
+    @Override
+    public String getUser() {
+        return SQLProperty.USER.getString(properties);
+    }
+
+    @Override
+    public void setUser(String userName) {
+        SQLProperty.USER.setString(properties, userName);
+    }
+
+    @Override
+    public String getPassword() {
+        return SQLProperty.PASSWORD.getString(properties);
+    }
+
+    @Override
+    public void setPassword(String password) {
+        SQLProperty.PASSWORD.setString(properties, password);
+    }
+
+    @Override
+    public String getDescription() {
+        return description;
+    }
+
+    @Override
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    @Override
+    public String getDataSourceName() {
+        return name;
+    }
+
+    @Override
+    public void setDataSourceName(String name) {
+        this.name = name;
+    }
+
+    @Override
+    public String getSocketChannelProvider() {
+        return SQLProperty.SOCKET_CHANNEL_PROVIDER.getString(properties);
+    }
+
+    @Override
+    public void setSocketChannelProvider(String classFqdn) {
+        SQLProperty.SOCKET_CHANNEL_PROVIDER.setString(properties, classFqdn);
+    }
+
+    @Override
+    public int getRequestTimeout() throws SQLException {
+        return (int) TimeUnit.MILLISECONDS.toSeconds(SQLProperty.REQUEST_TIMEOUT.getInt(properties));
+    }
+
+    @Override
+    public void setRequestTimeout(int seconds) {
+        SQLProperty.REQUEST_TIMEOUT.setInt(properties, (int) TimeUnit.SECONDS.toMillis(seconds));
```

Comment:
Here I'm a bit tentative about the same things about terms. The property named query timeout in java.sql.Connection (I mean the getter and the setter). Should not we follow this?